recipes/usalign/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -exo pipefail
+
+readonly PROGRAMS=(
+    qTMclust
+    USalign
+    TMalign
+    TMscore
+    MMalign
+    se
+    pdb2xyz
+    xyz_sfetch
+    pdb2fasta
+    pdb2ss
+    NWalign
+    HwRMSD
+    cif2pdb
+    pdbAtomName
+    addChainID
+)
+
+if [[ $(uname -s) == "Darwin" ]]; then
+    export CXX="${BUILD_PREFIX}/bin/clang++"
+fi
+
+cd "${SRC_DIR}"
```

Comment:
This is not needed. This is the current directory anyway.

Reply:
OK, I will remove it.